sdk/servicebus/Azure.Messaging.ServiceBus/tests/SenderLiveTests.cs

```diff
@@ -88,6 +90,71 @@ await using (var scope = await ServiceBusScope.CreateWithTopic(enablePartitionin
             }
         }
 
+        [Test]
+        public async Task SenderCanSendAMessageBatch()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
+            {
+                await using var client = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString);
+                ServiceBusSender sender = client.GetSender(scope.QueueName);
+                using ServiceBusMessageBatch batch = await sender.CreateBatchAsync();
```

Comment:
Can we add a test where we expect the TryAdd to return false?
Also since TryAdd doesn't throw, should we be doing the Assert.That stuff that we are doing in the other tests that were updated to use batches, i.e.

`                    Assert.That(() => batch.TryAdd(message), Is.True, "A message was rejected by the batch; all messages should be accepted.");
`